src/Sylius/Behat/Page/Admin/Crud/IndexPage.php

```diff
@@ -14,7 +14,7 @@
 use Behat\Mink\Session;
 use Sylius\Behat\Page\ElementNotFoundException;
 use Sylius\Behat\Page\SymfonyPage;
-use Sylius\Behat\TableManipulatorInterface;
+use Sylius\Behat\Service\Accessor\TableManipulatorInterface;
```

Comment:
If we decide to move `TableManipulatorInterface` to `Accessor` namespace can we also rename it to `TableAccessorInterface` whit rest of related classes and calls according to #4511?